static/images/letter-template/hm-government-cmyk.svg

```diff
@@ -0,0 +1,423 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 21.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 85.9 29.2" style="enable-background:new 0 0 85.9 29.2;" xml:space="preserve">
+<style type="text/css">
+	.st0{fill:#407CCA;}
```

Comment:
This is RGB. Needs more investigating…

Reply:
![image](https://user-images.githubusercontent.com/355079/34684918-9c5a5a6e-f49e-11e7-9e28-c68355f8ac6b.png)